packages/importer/src/columns.ts

```diff
@@ -84,14 +114,17 @@ export const guessColumnTypes = <T>(
   if (columnValues.every(isEmail)) {
     return 'email';
   }
+  if (columnValues.some(isMultiple)) {
```

Comment:
A good option might be to just never guess `multiple`

Reply:
I'm okay not guessing it, if uploading with an existing schema works or passing the type explicitly works.